### DISCUSSION TITLE
Screener: set_body -> size = 10000

### DISCUSSION THREAD

**Discussion by 1860** [ranaroussi/yfinance#2165] [Q&A]
Hi all,

I want to use the query / screener to create a large overview of the stock market. Unfortunately, if I set the screener body to a size of more than 200 my request crashes. I would like to set the size to 10,000. What can I do?

PS: I use the following settings

from requests import Session
from requests_cache import CacheMixin, SQLiteCache
from requests_ratelimiter import LimiterMixin, MemoryQueueBucket
from pyrate_limiter import Duration, RequestRate, Limiter
class CachedLimiterSession(CacheMixin, LimiterMixin, Session):
   pass
session = CachedLimiterSession(limiter=Limiter(RequestRate(2, Duration.SECOND*5)),bucket_class=MemoryQueueBucket,backend=SQLiteCache("yfinance.cache"),)

yf.set_tz_cache_location("~/yfinance")

**Comment by ericpien** [comment]
Hi, please refer to: https://github.com/ranaroussi/yfinance/issues/2152#issuecomment-2499499627

The limit is set by Yahoo, over which we have no control.

**Reply by 1860** [reply]
Thanks!
